api/v1alpha1/ippool_webhook.go

```diff
@@ -96,14 +98,14 @@ func (c *IPPool) checkPoolBonds(old *IPPool) ([]IPAddressStr, []IPAddressStr) {
 	allocationOutOfBonds := []IPAddressStr{}
 	inUseOutOfBonds := []IPAddressStr{}
 	for _, address := range c.Spec.PreAllocations {
-		inBonds := c.isAddressInBonds(address)
+		inBonds := c.isAddressInBounds(address)
```

Comment:
I think the original term (Bond) was correct one that we should use here instead of Bound

Reply:
I was not very sure what bond meant here, I thought it was trying to check if the address is in the bounds of the pool.  I switched back to the original wording as suggested.

Reply:
There was a positive and negative test cases in the webhook test class, using the start and end in the spec. I just added another pair for using Subnet in the pool spec.

Reply:
Thank you!